padl/transforms.py

```diff
@@ -658,38 +663,71 @@ def _pd_unpack_argument(self, arg) -> bool:
             return self._pd_number_of_inputs > 1
 
         try:
-            parameters = self._pd_get_signature().values()
+            parameters = self._pd_get_signature.values()
         except ValueError:
             return False
         for param in parameters:
-            if param.kind in (
+            param_kind = param.kind
```

Comment:
why this?

Reply:
Small optimization where an attribute lookup `param.kind` is slower than accessing a local variable (e.g. `param_kind`). Since we use `param.kind` more than once I assigned it to a local variable especially since this function is called a lot when evaluating a transform.